Dockerfile

```diff
@@ -2,15 +2,19 @@ FROM debian:bookworm-20240211
 
 WORKDIR /app
 
-COPY ./target/x86_64-unknown-linux-musl/release/icp-off-chain-agent .
-
 RUN apt-get update \
     && apt-get install -y ca-certificates \
     && apt-get -y install curl \
     && apt-get -y install ffmpeg
 
 EXPOSE 50051
 
+ENV GOOGLE_SA_KEY=test
+ENV YRAL_METADATA_TOKEN="test_token"
+ENV QSTASH_CURRENT_SIGNING_KEY="test_key"
+ENV QSTASH_AUTH_TOKEN="test_token"
```

Comment:
these can't be in this Dockerfile since this is used to deploy on fly.io

Reply:
hmm makes sense. so where do we put the dummy env for a local docker build?

Reply:
sounds good